### DISCUSSION TITLE
SSL issues for custom subdomain

### DISCUSSION THREAD

**Discussion by 0xidkfa** [GitbookIO/community#276] [0. General bugs and questions]
I'm running into some bizzare issues trying to point a subdomain at my Gitbook... Would love some help as I'm not sure what else to try!

DNS: 
![Screenshot 2023-07-25 at 5 39 59 PM](https://github.com/GitbookIO/community/assets/118562806/f697cf0d-8aaa-443a-814a-59dd146babf6)

Gitbook Interface:
![Screenshot 2023-07-25 at 11 33 28 AM](https://github.com/GitbookIO/community/assets/118562806/e0ea0658-609c-4166-a1e9-28ed17f6825e)

When I visit `dev.abracadabra.money`, I'm currently getting a SSL error `ERR_SSL_VERSION_OR_CIPHER_MISMATCH`:

![Screenshot 2023-07-25 at 5 42 06 PM](https://github.com/GitbookIO/community/assets/118562806/256f741a-8fe3-4a20-ac4e-f462404790b0)

I then added the following CAA record, but it was no use:
![Screenshot 2023-07-25 at 5 40 44 PM](https://github.com/GitbookIO/community/assets/118562806/f0aedc64-399c-419b-a251-747b9d7136bd)

**Comment by rodrcastro** [comment]
Hey @0xidkfa, thank you for your message in our community!

Could you please email us at support@gitbook.com so we can take a closer look at this? Someone from the support team will guide you through this. Thank you!

> Disclaimer: Please note that this is not our official support channel. If you encounter any bugs or issues, we strongly encourage you to contact [support@gitbook.com](mailto:support@gitbook.com).
